utils/bump_package_versions.js

```diff
@@ -20,10 +20,12 @@ const path = require('path');
 const fs = require('fs');
 const { execSync } = require('child_process');
 
-const { packages } = require('./list_packages.js');
+const { packages, packagesToPublish } = require('./list_packages.js');
 
 (async () => {
   const version = process.argv[2];
+  if (version.startsWith('v'))
+    throw new Error('Version must not start with "v"');
```

Comment:
I guess it would be better to move this after the existing `if (!version)` check. Otherwise it might get a reference error.

Reply:
very fair; thank you for pointing out! https://github.com/microsoft/playwright/pull/11448